Blog/db/seeds.rb

```diff
@@ -0,0 +1,16 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+Post.destroy_all
```

Comment:
Just a word of warning here, this makes `db:seed` a destructive action. You would probably not want to allow this to run in a production environment for example.